### ISSUE TITLE
CHECK_MINIMAL_CXX_STANDARD raises warning since macro is not available

### ISSUE THREAD

**Issue by wxmerkt** [jrl-umi3218/jrl-cmakemodules#478]
I am using projects builds with `jrl-cmakemodules` in downstream projects that do not use the `jrl-cmakemodules`. Hence, the lines added to check minimal C++ standards in the `projectConfig.cmake` do not check the version and instead raise a warning (the macro isn't included/available):

```cmake
if(COMMAND CHECK_MINIMAL_CXX_STANDARD)
  CHECK_MINIMAL_CXX_STANDARD(14)
else()
  MESSAGE(WARNING "This dependency requires C++ >= 14")
endif()
```

Is there a recommended solution? E.g. to include the macro in the generated `projectConfig`?

**Comment by wxmerkt** [comment]
Are other macros used in the generated `Config`? If not, we can just fix the exported part in `package-config.cmake`

**Comment by nim65s** [comment]
This macro requires a c++ file to work.

We also need to ship stuff in find-external when they are used, ref. eg. https://github.com/loco-3d/ndcurves/pull/50

**Comment by wxmerkt** [comment]
Thanks for the insight - I've been working on a potential solution locally and will continue Monday and keep you posted

**Comment by wxmerkt** [comment]
Resolved via #479
